tests/lib.rs

```diff
@@ -16,6 +16,31 @@ fn levenshtein_works() {
     assert_eq!(3, levenshtein("kitten", "sitting"));
 }
 
+#[test]
+fn normalized_levenshtein_works() {
+    assert!((normalized_levenshtein("kitten", "sitting") - 0.57142).abs() < 0.00001);
+}
+
+#[test]
```

Comment:
This is another nitpick, but would you mind only leaving one test for each function in this file? These tests are integration tests that only need to verify that the functions can be called from the outside. You can move the rest of the tests to the bottom of `src/lib.rs` to serve as unit tests.

Reply:
Makes sense. Will do as you suggested :+1:

Reply:
Moved  tests in 8f4c33759af0ea84e77f467df306dea5ef727f11